.circleci/config.yml

```diff
@@ -111,6 +111,41 @@ workflows:
             branches:
               only: master
 
+  # Separate workflow just for version tag releases.
+  tagged-release:
+    jobs:
+      - test:
+          filters: &version-tag-only
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*/
+
+      - test_full:
+          filters: *version-tag-only
+
+      - test_full_10:
+          filters: *version-tag-only
+
+      - docker/publish:
+          filters: *version-tag-only
+          requires:
+            - test
+            - test_full
+            - test_full_10
+          image: sourcecred/sourcecred
+          tag: latest
+          extra_build_args: --cache-from=node:12,sourcecred/sourcecred:dev
```

Comment:
could this ever be node 10? And just to double check - the dev tag is going to be the latest (and likely most useful for cache, moreso than latest?)

Reply:
Currently we only ship `node:12` based images (See [`Dockerfile`](https://github.com/sourcecred/sourcecred/blob/279bbdd451d4fec23b44baf16231b65d3ed7db36/Dockerfile#L1)).
We do use `node:10` for tests, but that's a Circle executor, not our base image.

Reply:
okay doke, this sounds good to me.